lib/manageiq/appliance_console/database_admin.rb

```diff
@@ -140,21 +138,22 @@ def ask_ftp_file_options
       end
 
       def ask_custom_file_options(server_uri)
-        @filename    = just_ask(*filename_prompt_args) unless action == :restore
-        sample_case  = server_uri.split("/").last
-        hostname     = URI(server_uri).host
-        uri_filename = ask_custom_prompt(hostname, 'filename', "Target filename (e.g.: #{sample_case})")
-        @uri         = server_uri.gsub(sample_case, uri_filename)
+        hostname  = URI(server_uri).host
+        @filename = ask_custom_file_prompt(hostname)
+        @uri      = server_uri
 
-        params = { :uri => uri }
-        params[:remote_file_name] = filename if filename
+        params = {
+          :uri              => uri,
+          :skip_directory   => true,
```

Comment:
I am fine with this for now, but I think in the end, this should be part of the configuration, and not hard coded in.

Reply:
Along with this, I really feel we should be hiding this option, or allowing it to be configured to do so, for the restore options since this is ultimately being implemented for an endpoint you won't be able to restore from.

Might need a follow up, but just my two cents.